picasso/src/test/java/com/squareup/picasso3/DispatcherTest.java

```diff
@@ -263,7 +265,7 @@
   }
 
   @Test public void performErrorCleansUpAndPostsToMain() {
-    BitmapHunter hunter = mockHunter(URI_KEY_1, new RequestHandler.Result(bitmap1, MEMORY));
+    BitmapHunter hunter = mockHunter(URI_KEY_1 + KEY_SEPARATOR, new RequestHandler.Result(bitmap1, MEMORY));
```

Comment:
why append KEY_SEPARATOR now?

Reply:
Because now the getKey method is not mocked. It uses the real Request's key.
When the key is built with a user-set stable key prefix, it appends the spearator onto the key.

Reply:
All very unfortunate.

Reply:
That's the phrase I use to describe almost all of Picasso's tests!